core/src/main/java/io/questdb/TelemetryConfigLogger.java

```diff
@@ -68,6 +84,31 @@ public void close() {
         configWriter = Misc.free(configWriter);
     }
 
+    @Override
+    public void update(PreferencesMap preferencesMap) {
+        instanceName.clear();
+        instanceName.put(preferencesMap.get(INSTANCE_NAME));
+        instanceType = preferencesMap.get(INSTANCE_TYPE);
+        instanceType = instanceType == null ? "" : instanceType;
+        instanceDesc.clear();
+        instanceDesc.put(preferencesMap.get(INSTANCE_DESC));
+
+        try (final SqlCompiler compiler = engine.getSqlCompiler()) {
+            final SqlExecutionContextImpl sqlExecutionContext = new SqlExecutionContextImpl(engine, 1);
+            sqlExecutionContext.with(
+                    engine.getConfiguration().getFactoryProvider().getSecurityContextFactory().getRootContext(),
+                    null,
+                    null
+            );
+            updateTelemetryConfig(engine, compiler, sqlExecutionContext, configTableToken);
+        } catch (Throwable th) {
+            LOG.error()
+                    .$("could not update config telemetry [table=`").utf8(TELEMETRY_CONFIG_TABLE_NAME)
+                    .$("`, msg=").$(th.getMessage())
```

Comment:
do not log `th.getMessage()` when catching throwable - log the whole exception. NPE will produce "null" message and no stack trace

Reply:
addressed in https://github.com/questdb/questdb/pull/5689/commits/f9eb453c28e538ac2401e7fd7b44b2bc172a0d82